datasetconfig.json

```diff
@@ -492,14 +511,15 @@
         "cache": 6,
         "climatology": "http://navigator.oceansdata.ca:8080/thredds/dodsC/climatology/Levitus98_PHC21/aggregated.ncml",
         "attribution": "The Canadian Centre for Meteorological and Environmental Prediction",
-        "lat_var_key": "y",
-        "lon_var_key": "x",
+        "lat_var_key": "nav_lat",
+        "lon_var_key": "nav_lon",
         "help": " ",
 
         "variables": {
-            "somecrty": { "name": "ocean current along j-axis", "envtype": "ocean", "unit": "m/s", "scale": [-1, 1], "zero_centered": "true" },
-            "sozocrtx": { "name": "ocean current along i-axis", "envtype": "ocean", "unit": "m/s", "scale": [-1, 1], "zero_centered": "true" },
-            "magwatervel": { "name": "ocean current magnitude", "envtype": "ocean", "unit": "m/s", "scale": [0, 1], "equation": "magnitude(sozocrtx, somecrty)",  "dims": ["time_counter", "y", "x"] },
+            "sozocrtx": { "name": "Water East Velocity", "envtype": "ocean", "unit": "m/s", "scale": [-1, 1], "zero_centered": "true" },
```

Comment:
sozocrtx should be hidden (or removed). And the name should be "ocean current along i-axis"

Reply:
I do believe that there is a way to have a variable hidden from view. I just need to look at a much older datasetconfig.json example. If this works I'll implement it and test. If they work as desired, I will then reach out via another update of my branch and will look at implementing the datasetconfig.json in a testing container. If my interactions provide the desired outcome. I will then ask for you to participate and if your satisfied with your interactions with the site. I will then seek to have the work committed.